projects/BFDR/IJEFetalDeath.cs

```diff
@@ -244,7 +244,7 @@ public string AUXNO
             {
                 if (String.IsNullOrWhiteSpace(record?.StateLocalIdentifier1))
                 {
-                    return "".PadLeft(12, '0');
+                    return (new String(' ', 12));
                 }
                 string auxNo = record.StateLocalIdentifier1;
                 if (auxNo.Length > 12)
```

Comment:
the line auxNo.PadLeft(12, '0') below should be changed to use LeftJustifiedGet, same as we do in VRDR https://github.com/nightingaleproject/vrdr-dotnet/blob/b722e4101e566c05b0721b4e266858718338f3db/VRDR/IJEMortality.cs#L1099

Reply:
and the RightJusitifiedZeroed_Set below should be updated to to use LeftJustifiedSet same as in VRDR https://github.com/nightingaleproject/vrdr-dotnet/blob/b722e4101e566c05b0721b4e266858718338f3db/VRDR/IJEMortality.cs#L1105

Reply:
Modified to match VRDR code on get and set.